navigation/ca_move_base/launch/navigate.launch

```diff
@@ -3,13 +3,21 @@
   <arg name="laser"     value="$(optenv LASER rplidar)" doc="Laser for mapping the environment"/>
 
   <arg name="localization" doc="Autonomous method for localization/navigation."/>
```

Comment:
Can you remove [this argument](https://github.com/RoboticaUtnFrba/create_autonomy/blob/kinetic-devel/ca_gazebo/launch/spawn_multirobot.launch#L6-L7) from there and use it here?

Reply:
@eborghi10 you mean the `optenv` for localization?

Reply:
@eborghi10 you mean the optenv for localization?

Reply:
Yes, because as you can see in that launch file, the argument `localization` is passed to `navigate.launch` (this launch file) and to itself. So there is no need to declare it there. It can be done here.

Reply:
Done!

Reply:
@eborghi10 and what about the check that `spawn_multirobot` does to see if should launch `navigate` or not? It depends on the value of `localization`.

Reply:
It could be done internally with a giant `group if` but don't worry. Leave it as it is.